### DISCUSSION TITLE
Lab2 tiny error in workbook page8

### DISCUSSION THREAD

**Discussion by Lin-Xiang88** [bol-edu/HLS-SOC-Discussions#64] [General]
2022.1-Workbook-Lab2-KV260.pdf page8
(KV260 檔案位置為 ol = Overlay("/home/root/jupyter_notebook/FIRN11Stream.bit"))
應該要改成
(KV260 檔案位置為 ol = Overlay("/home/root/jupyter_notebooks/FIRN11MAXI.bit"))
jupyter_notebooks少加s

**Comment by boledulab** [comment]
已更新到文件
